lib/omnibus/packagers/ociru.rb

```diff
@@ -0,0 +1,282 @@
+#
+# Copyright 2014 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "find"
+require "json"
+require "pathname"
+require "omnibus/packagers/windows_base"
+require "fileutils"
+
+module Omnibus
+  class Packager::OCIRU < Packager::Base
+    id :ociru
+    intermediate_pkg_name = "package.tar.xz"
+
+    setup do
+    end
+
+    build do
+      # create the payload directory, copy the install_dir and extra files to it
+      # TODO: does the makedirs do what we need in terms of file permissions?
```

Comment:
should you use `create_sync_dir` then?

Reply:
indeed! Done

Reply:
Ah now I remember! `FileSyncer.sync`already calls `create_sync_dir` so I just removed the `TODO`